tests/integration/components/light-table/columns/base-test.js

```diff
@@ -11,7 +11,46 @@ module('Integration | Component | Columns | base', function (hooks) {
     // Set any properties with this.set('myProperty', 'value');
     // Handle any actions with this.on('myAction', function(val) { ... });"
     this.set('column', Column.create());
-    await render(hbs`{{light-table/columns/base column=this.column}}`);
+    await render(hbs`<LightTable::Columns::Base @column={{this.column}} />`);
     assert.dom('*').hasText('');
   });
+
+  test('column alignment', async function (assert) {
```

Comment:
@maxwondercorn this doesn't test anything because we're rendering a new component each time. What I was concerned about was the alignment changing on the column.

We should render one time, then change the alignment on the column and make sure the correct alignment class is applied.

Reply:
@rwwagner90 I wasn't thinking of these as dynamic properties probably because I don't use them that way in my application.  Since they were computed properties they would have been dynamic and they still need to be.... 🤦 

I already changed `isSorted` to computed because it was not working.  Will update the other's and tests

Reply:
Fixed by adding @compute to `column.align`